docs/website/docs/hub/init.md

```diff
@@ -0,0 +1,301 @@
+---
+title: Initialize a pipeline
+description: How to initialize and develop a dlt pipeline using CLI, workspace, or verified sources
+keywords: [create a pipeline, verified source, workspace, rest client, how to, dlt init]
+---
+
+# Initialize a pipeline
+
+This guide walks you through creating and initializing a `dlt` pipeline — whether manually via the CLI, through the LLM-native **dltHub workspace**, or from one of the community-maintained **verified sources**.
+
+## Overview
+
+A `dlt` pipeline moves data from a source (like an API or database) into a destination (like DuckDB, Snowflake, or Iceberg).
+You can create one in three ways:
+
+| Method | Command | Best for |
+|---------|----------|----------|
+| CLI-based | `dlt init <source> <destination>` | Developers who prefer manual setup |
+| Workspace-based | `dlt init dlthub:<source> <destination>` | AI-assisted development with editors like Cursor |
+| Verified source | `dlt init <verified_source> <destination>` | Prebuilt, tested connectors from the community |
```

Comment:
What about `core sources` should we also talk about 
`dlt init sql_database duckdb` and such?